contextualSpellCheck/contextualSpellCheck.py

```diff
@@ -1,15 +1,33 @@
 import spacy
 import torch
 import editdistance
-import datetime
+from datetime import datetime
 import os
 import copy
 import warnings
 
 from spacy.tokens import Doc, Token, Span
 from spacy.vocab import Vocab
 
-from transformers import AutoModelWithLMHead, AutoTokenizer
+from transformers import AutoModelForMaskedLM, AutoTokenizer
+
+
+def token_require_spell_check(token):
+    """
+    Getter for Token attributes.
+
+    Args:
+        token {`Spacy.Token`} :Token object for the value should be returned
+
+    Returns:
+        List :If no suggestions: False else: True
+    """
+    return any(
+        [
+            token.i == suggestion.i and token.text == suggestion.text
+            for suggestion in token.doc._.suggestions_spellCheck.keys()
+        ]
+    )
```

Comment:
How is this different from the original function at [L360 in master and L428 in this PR](https://github.com/R1j1t/contextualSpellCheck/blob/master/contextualSpellCheck/contextualSpellCheck.py#L360-L374)?

Reply:
Thank you for pointing out  this. That's my bad. I duplicated this function.

Reply:
Removed duplicated function.